.github/workflows/docker-push.yml

```
@@ -0,0 +1,43 @@
+name: Deploy IOTA Operator on DockerHub
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'develop'
+
+  workflow_dispatch:
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          username: ${{ secrets.IOTALEDGER_DOCKER_USERNAME }}
+          password: ${{ secrets.IOTALEDGER_DOCKER_PASSWORD }}
+      
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: iotaledger/wasp
+      
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: 
+            - BUILD_TAGS=rocksdb,builtin_static
+            - BUILD_LD_FLAGS=-X github.com/iotaledger/wasp/packages/wasp.VersionHash=${{ env. GITHUB_SHA }}
```

```suggestion
          build-args: BUILD_TAGS=rocksdb,builtin_static,BUILD_LD_FLAGS=-X github.com/iotaledger/wasp/packages/wasp.VersionHash=${{ env. GITHUB_SHA }}
```
The documentation is not great. It is a CSV List: https://github.com/docker/build-push-action#inputs